src/object/mod.rs

```diff
@@ -1,4 +1,36 @@
 //! Traits for working with JavaScript objects.
+//!
+//! This module defines the [`Object`](Object) trait, which is implemented
+//! by all object types in the [JavaScript type hierarchy][hierarchy]. This
+//! trait provides key operations in the semantics of JavaScript objects,
+//! such as getting and setting an object's properties.
+//!
+//! ## Property Keys
+//!
+//! Object properties are accessed by a _property key_, which in JavaScript
+//! can be a string or [symbol][symbol]. (Neon does not yet have support for
+//! symbols.) For convenience, the [`PropertyKey`](PropertyKey) trait allows
+//! Neon programs to use various Rust string types, as well as numeric types,
+//! as keys when accessing object properties, converting the keys to strings
+//! as necessary:
+//!
+//! ```
```

Comment:
This needs a `cfg` to skip it in non-N-API and the `get` and `set` functions need a context added.

Reply:
Fixed and CI is passing again!